### PR TITLE
[TS-SELENIUM] Create class for interaction with workspace API

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Create a class for interaction with workspace API

### What issues does this PR fix or reference?
issue: https://github.com/eclipse/che/issues/13291

### CODE REVIEW

**Review by musienko-maxim — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
ci-build

File: e2e/package.json [hunk 1]
```diff
@@ -13,12 +13,12 @@
     "@types/mocha": "^5.2.6",
     "@types/node": "^11.13.4",
     "@types/selenium-webdriver": "^3.0.16",
+    "axios": "^0.18.0",
```
Comment (monaka) [inline e2e/package.json]: Missing CQ? https://dev.eclipse.org/ipzilla/buglist.cgi?quicksearch=axios (refs https://github.com/eclipse/che/pull/13207#pullrequestreview-233808707)

**Review by monaka — CHANGES_REQUESTED** [review]

File: e2e/package.json [hunk 2]
```diff
@@ -13,16 +13,16 @@
...
-    "chai": "^4.2.0",
+    "axios": "^0.17.1",
+    "chai": "^4.1.2",
     "chromedriver": "^2.46.0",
     "mocha": "^6.1.4",
     "selenium-webdriver": "^3.6.0",
-    "ts-node": "^8.0.3",
-    "typed-rest-client": "^1.2.0",
-    "typescript": "^3.4.3"
+    "ts-node": "^8.1.0",
```
Comment (monaka) [inline e2e/package.json:21]: ts-node@8.1.0 is still `awaiting` now. https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19663     I think no fix is required. Just waiting until fixed CQ.

File: e2e/package.json [hunk 3]
```diff
@@ -13,16 +13,16 @@
...
+    "typescript": "^3.4.5"
```
Comment (monaka) [inline e2e/package.json:22]: This is same as ts-node@8.1.0 . CQ check will be passed with no issue. Just waiting now.

**Review by monaka — COMMENTED** [review]
LGTM after CQ requests are fixed.

File: e2e/package.json (hunk 2 above)
Comment (monaka) [reply e2e/package.json:21]: It was passed CQ check https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19663. So I mark this conversation resolved.

File: e2e/package.json (hunk 3 above)
Comment (monaka) [reply e2e/package.json:22]: Tracking https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19662

File: e2e/package.json (hunk 2 above)
Comment (Ohrimenko1988) [reply e2e/package.json:21]: Thank you for your attention.

File: e2e/package.json (hunk 3 above)
Comment (monaka) [reply e2e/package.json:22]: The CQ was resolved by license_cerified. So I mark this conversation resolved.

**Comment by monaka** [comment]
LGTM as CQ passed.

@Ohrimenko1988 Could you resolve some conflicts and re-push ?

**Review by monaka — APPROVED** [review]

**Pull request closed without merging**